### PR TITLE
Content and terminology tweaks in guides

### PR DESCRIPTION
**Pull request by BBashh**
Made some changes in the guides folder.

Improved readability and fixed typos.

### CODE REVIEW

File: src/content/guides/how-to-create-an-ethereum-account/index.md [hunk 1]
```diff
@@ -51,17 +51,17 @@ Once you have saved your recovery phrase you should see your wallet dashboard wi
 
 No, just like online banking you can have many different accounts all stored in one wallet application. Your 12 or 24 word phrase secures them all: it is like the seed of a big tree (which is why you must keep it safe). Each branch of the tree holds a key, and every key is one of your accounts. If you lose access to the wallet (i.e. your tree gets cut down), you can always restore all your different accounts using other software and the same seed (which will always grow the same tree).
 
-### Can I send bitcoin to an Ethereum address, or ether to a Bitcoin address?
+### Can I send Bitcoin to an Ethereum address, or Ether to a Bitcoin address?
```
Comment (minimalsm) [inline src/content/guides/how-to-create-an-ethereum-account/index.md:54]:
```suggestion
### Can I send bitcoin to an Ethereum address, or ether to a Bitcoin address?
```

Uppercase 'Bitcoin' refers to the network, bitcoin (the currency) uses a lowercase b, unless at the beginning of a sentence. Same rule for ether.

File: src/content/guides/how-to-create-an-ethereum-account/index.md [hunk 2]
```diff
@@ -51,17 +51,17 @@ Once you have saved your recovery phrase you should see your wallet dashboard wi
...
 
-No you cannot. Bitcoin and ether exist on two separate networks (i.e. different blockchains), each with their own bookkeeping models and address formats. There have been various attempts to bridge the two different networks, of which the most active one is currently [Wrapped bitcoin or WBTC](https://www.bitcoin.com/get-started/what-is-wbtc/). This is not an endorsement, as WBTC is a custodial solution (meaning a single group of people controls certain critical functions) and is provided here for informational purposes only.
+No, you cannot. Bitcoin and Ether exist on two separate networks (i.e. different blockchains), each with their own bookkeeping models and address formats. There have been various attempts to bridge the two different networks, of which the most active one is currently [Wrapped Bitcoin or WBTC](https://www.bitcoin.com/get-started/what-is-wbtc/). This is not an endorsement, as WBTC is a custodial solution (meaning a single group of people controls certain critical functions) and is provided here for informational purposes only.
```
Comment (minimalsm) [inline src/content/guides/how-to-create-an-ethereum-account/index.md:56]: Per above 

```suggestion
No, you cannot. Bitcoin and ether exist on two separate networks (i.e. different blockchains), each with their own bookkeeping models and address formats. There have been various attempts to bridge the two different networks, of which the most active one is currently [Wrapped bitcoin or WBTC](https://www.bitcoin.com/get-started/what-is-wbtc/). This is not an endorsement, as WBTC is a custodial solution (meaning a single group of people controls certain critical functions) and is provided here for informational purposes only.
```

File: src/content/guides/how-to-use-a-wallet/index.md [hunk 3]
```diff
@@ -22,7 +22,7 @@ Your address (which is sometimes called a "public key") is like your home addres
 
 You need to provide whoever wants to send you money with your public address. Many wallet apps let you copy your address or show a QR code to scan for easier usage. Avoid typing any Ethereum address manually. This can easily lead to clerical errors and lost funds.
 
-Different apps may vary or use different language, but they should take you through a similar process if you are trying to transfer funds.
+Different apps may vary or use different languages, but they should take you through a similar process if you are trying to transfer funds.
```
Comment (minimalsm) [inline src/content/guides/how-to-use-a-wallet/index.md:25]: Language here refers to the terminology/phrasing they use, not the communication system they use (e.g. English, French, etc). 

cc: @konopkja maybe this is confusing?

```suggestion
Different apps may vary or use different language, but they should take you through a similar process if you are trying to transfer funds.
```

**Review by minimalsm — CHANGES_REQUESTED** [review]
Hey @BBashh, thanks for the PR ❤️. A few suggested changes (reversions) and this is good to go 🫡

File: src/content/guides/how-to-use-a-wallet/index.md (hunk 3 above)
Comment (konopkja) [reply src/content/guides/how-to-use-a-wallet/index.md:25]: agree, a clearer "wording" should be used

File: src/content/guides/how-to-create-an-ethereum-account/index.md (hunk 1 above)
Comment (BBashh) [reply src/content/guides/how-to-create-an-ethereum-account/index.md:54]: Okay, got it. Thanks for the information.

**Comment by BBashh** [comment]
Thanks, @minimalsm for adding me as a contributor for content. I will continue to contribute with all my dedication and determination.

**Pull request merged**